### PR TITLE
Use new semantic color names (info, confirm, notify and alert)

### PR DESCRIPTION
**Pull request by amelako**
## Purpose

In order to match the new naming of colors and variants in the Notification components, added info, confirm, notify and alert colors to design tokens.

## Approach and changes

Added info, alert, notify and confirm types in design tokens.
Updated light.ts with the new colors.
Replaced the warning, success and danger color usages in circuit ui with the new ones.

## Definition of done

* [ ] Development completed
* [ ] Reviewers assigned
* [ ] Unit and integration tests
* [ ] Meets minimum browser support
* [ ] Meets accessibility requirements

### CODE REVIEW

**Review by amelako — COMMENTED** [review]
@robinmetral updated most of the places in circuit ui as well. 
There are few things I would like to clear up:
1. Old Notification component and InlineMessage have the danger/success/warning variants. I did update the colors, which now contradicts with the naming a bit. But I don't see the point of changing the variant naming to match since these are gonna be deprecated anyway. Let me know if you agree?
2. Body component has variants success and error as well as Badge success, warning and danger. I just changed the colors usage there to match the new ones. Let me know if you have any thoughts on the variants cause that would mean a bigger change then.
Let me know if you have any other thoughts or ideas. Thank :)

**Comment by robinmetral** [comment]
> 1. Old Notification component and InlineMessage have the danger/success/warning variants. I did update the colors, which now contradicts with the naming a bit. But I don't see the point of changing the variant naming to match since these are gonna be deprecated anyway. Let me know if you agree?

Makes sense, let's keep it 👍 

> 2. Body component has variants success and error as well as Badge success, warning and danger. I just changed the colors usage there to match the new ones. Let me know if you have any thoughts on the variants cause that would mean a bigger change then.

Ah, that's true. Let's keep it for now and handle the renaming in the next major (maybe you can add a comment in there to remind us).

**Review by connor-baer — APPROVED** [review]

**Comment by connor-baer** [comment]
>> Body component has variants success and error as well as Badge success, warning and danger. I just changed the colors usage there to match the new ones. Let me know if you have any thoughts on the variants cause that would mean a bigger change then.
> 
> Ah, that's true. Let's keep it for now and handle the renaming in the next major (maybe you can add a comment in there to remind us).

You could add the new variants already and deprecate the old ones, would make the migration easier.

**Comment by robinmetral** [comment]
> > > Body component has variants success and error as well as Badge success, warning and danger. I just changed the colors usage there to match the new ones. Let me know if you have any thoughts on the variants cause that would mean a bigger change then.
> > 
> > 
> > Ah, that's true. Let's keep it for now and handle the renaming in the next major (maybe you can add a comment in there to remind us).
> 
> You could add the new variants already and deprecate the old ones, would make the migration easier.

That's true—let's handle it in a separate PR to keep this one's scope small 🙂

**Comment by robinmetral** [comment]
> > > > Body component has variants success and error as well as Badge success, warning and danger. I just changed the colors usage there to match the new ones. Let me know if you have any thoughts on the variants cause that would mean a bigger change then.
> > > 
> > > 
> > > Ah, that's true. Let's keep it for now and handle the renaming in the next major (maybe you can add a comment in there to remind us).
> > 
> > 
> > You could add the new variants already and deprecate the old ones, would make the migration easier.
> 
> That's true—let's handle it in a separate PR to keep this one's scope small 🙂

Nested quote replies FTW

File: packages/design-tokens/themes/light.ts [hunk 1]
```diff
@@ -105,6 +105,10 @@ const misc = {
   danger: '#D23F47',
   success: '#138849',
   warning: '#F5C625',
```
Comment (robinmetral) [inline packages/design-tokens/themes/light.ts:107]: Remember to mark these as deprecated

**Review by robinmetral — COMMENTED** [review]
Also needs a couple changesets (for Circuit and the tokens), but looks good to go ✨ 

\+ I like Connor's idea of immediately adding the new variant names to the `Body`, `BodyLarge` and `Badge`, and deprecating the old variant names. Let's split it into a separate PR

++ You could also make this another entry in the draft migration guide from v4.x to v5 (since that's when we'll remove the old variant names)

File: packages/design-tokens/themes/light.ts (hunk 1 above)
Comment (amelako) [reply packages/design-tokens/themes/light.ts]: Done in [6f6a687](https://github.com/sumup-oss/circuit-ui/pull/1431/commits/6f6a687dea9e1d2a955b6c9d428939ffadff9714). Let me know if there is something else needed.

File: .changeset/seven-books-tan.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/design-tokens': minor
+---
+
+Added a new `info`, `confirm`, `alert` and `notify` colors and deprecated the `success`, `warning` and `danger`.
```
Comment (connor-baer) [inline .changeset/seven-books-tan.md:5]:
```suggestion
Added new `info`, `confirm`, `alert`, and `notify` colors and deprecated the `success`, `warning`, and `danger` colors.
```

File: .changeset/wicked-timers-bake.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/circuit-ui': minor
+---
+
+Replaced the `success`, `warning` and `danger` color usage with the new values `confirm`, `notify` and `alert`.
```
Comment (connor-baer) [inline .changeset/wicked-timers-bake.md:5]:
```suggestion
Replaced the `success`, `warning`, and `danger` colors with the new `confirm`, `notify`, and `alert` colors.
```

**Review by connor-baer — APPROVED** [review]

**Review by robinmetral — APPROVED** [review]
💯

**Comment by robinmetral** [comment]
The Chromatic change is unexpected, I'm re-running the job to see if there was a hiccup

**Pull request merged**